src/VirtualFile.php

```diff
@@ -225,4 +226,15 @@ public function createPublicLink($config = [], $fetch = PublicLink::OBJECT)
 
         return $link;
     }
-}
\ No newline at end of file
+
+    /**
+     * @param EventReport $eventReport
+     *
+     * @return array
+     */
+    public function getEvents(EventReport $eventReport)
+    {
+        $eventsRaw = $this->client->FileEventReport($eventReport, $this->getShareId(), $this->getInternalName(), $this->domain, $this->token);
+        return Collection::make($eventsRaw);
```

Comment:
`Collection` is a helper class from Laravel for working with arrays. Although we use it to keep properties and other arrayable data, we always return raw array. I've done so not to introduce a new class to the outside scope. Returning a Collection instance won't cause any error as it implements the same interface as array, but for consistency I think it's better to return just raw data here. Or abstract it into separate class.

```suggestion
        return $eventsRaw;
```